doc/internals.rst

```diff
@@ -116,6 +116,10 @@ xarray:
 
 .. literalinclude:: examples/_code/accessor_example.py
 
+In general, the only requirement is that the accessor's ``__init__`` only has a single
+parameter except from ``self``: the ``Dataset`` or ``DataArray`` object it is supposed
+to work on.
```

Comment:
thanks for the hint, @mathause, I didn't notice that when writing it. How about this:
```suggestion
In general, the only restriction on the accessor class is that the ``__init__`` method
has just a single parameter: the ``Dataset`` or ``DataArray`` object it is supposed
to work on.
```

Reply:
Including @dcherian's suggestion, maybe something like this:
```suggestion
In general, the only restriction on the accessor class is that the ``__init__`` method
must have a single parameter: the ``Dataset`` or ``DataArray`` object it is supposed
to work on.
```

Reply:
The second one looks good.